libs/pages/logs/application/src/lib/ui/row/row.tsx

```diff
@@ -0,0 +1,78 @@
+import { Log } from 'qovery-typescript-axios'
+import { CopyToClipboard, Icon, IconAwesomeEnum } from '@qovery/shared/ui'
+import { dateFullFormat } from '@qovery/shared/utils'
+
+const COLORS = [
+  '#FFF',
```

Comment:
I will like the white color should not be part of the list. It is weird to sometime has a pod that is white. You wonder why this one has the same color as the rest of the message? I found it disturbing personally.

Reply:
Yes I agree, I'll add another color to replace it !